envoy.dependency.pip_check/envoy/dependency/pip_check/abstract.py

```diff
@@ -102,3 +101,10 @@ def dependabot_success(self, correct: Iterable) -> None:
     def dependabot_errors(self, missing: Iterable, msg: str) -> None:
         for dirname in sorted(missing):
             self.error("dependabot", [f"{msg}: {dirname}"])
+
+    def dir_matches(self, path: pathlib.Path) -> bool:
+        """For given file path, check if its a requirements file and whether
+        its parent directory is excluded."""
+        return (path.name == self.requirements_filename and
```

Comment:
for multi-line conditions please place the conjoining statements - `and`, `not`, `or` etc at the beginning of the line - ie

```python
    def dir_matches(self, path: pathlib.Path) -> bool:
        """For given file path, check if its a requirements file and whether
        its parent directory is excluded."""
        return (
            path.name == self.requirements_filename
            and not self.ignored_dirs.match(
                f"/{path.parent.relative_to(self.path)}"))
```

The pep8 spec on this used to suggest that these should be at the line end, and i have always disabled that check - now i think they have flipped it